kpi/exceptions.py

```diff
@@ -62,6 +62,12 @@ class KobocatDuplicateSubmissionException(exceptions.APIException):
     default_code = 'submission_duplication_error'
 
 
+class KobocatBulkUpdateSubmissionsException(exceptions.APIException):
+    status_code = 500
```

Comment:
when `KobocatBulkUpdateSubmissionsException` is raised by `__validate_payload()`—if the messages used there are to be returned to the client—would the client receive a 500?

if a client sends incorrect data to the API, they should receive a 400. please reserve 500 errors for situations where it's us as the developers / sysadmins who are responsible for fixing the problem, i.e. a bug in the code or a server-side system not working properly

Reply:
Thanks for pointing this out! I've created a new exception class to differentiate client and server-side errors 👍